addons/pos_restaurant/static/src/app/floor_screen/table.js

```diff
@@ -70,11 +90,13 @@ export class Table extends Component {
     }
     get orderCount() {
         const table = this.props.table;
-        return table.order_count !== undefined
-            ? table.order_count
-            : this.pos.globalState
-                  .getTableOrders(table.id)
-                  .filter((o) => o.orderlines.length !== 0 || o.paymentlines.length !== 0).length;
+
+        if (table.changes_count > 0) {
+            return table.changes_count;
+        } else if (table.skip_changes > 0) {
+            return table.skip_changes;
+        }
+        return table.order_count;
```

Comment:
With the move of ` await this.pos.globalState._syncTableOrdersToServer();` this function should maybe be modified to pass tests. Will see..